spec/tags/subscriptions.md

```diff
@@ -33,6 +34,7 @@ The <code>description</code> field is directly tied to the <code>code</code> fie
 * *READ: "**The message has been read by the recipient**".*
 * *REJECTED: "**The message was rejected by the provider**".*
 * *NOT_DELIVERED: "**The provider was unable to deliver the message**".*
+* *READ: "**The message has been deleted by the recipient**".*
```

Comment:
Acho que precisa ajustar aqui para "DELETED"

Reply:
esquece kkkkk entendi agora

Reply:
e pensando bem agora, acho que a descrição poderia ser "by the sender" em vez de recipient, pra deixar claro que é de MO o status.

Reply:
ou "by the contact" se quiser deixar genérico pro caso de algum canal dar pra deletar o MT também.